### DISCUSSION TITLE
Asking for new ideas about adding example nextjs with react-flow

### DISCUSSION THREAD

**Discussion by igihcksn** [vercel/next.js#17405] [Ideas]
Hi all,

i'm new in react or nextjs and i'm getting project that must use diagram flow like [react-flow](https://github.com/wbkd/react-flow).

before doing to use that library i'm doing research about how to use, and implement in nextjs project, but when im searching on exmple in this repo i didn't see example that use react-flow library from [wbkd](https://github.com/wbkd) and i think why don't i just do this, but the big question is, may i doing request to adding an example to use nextjs with react-flow?

sory about my bad english before, tahanks and i hope that you all understand what i mean

**Comment by lfades** [comment]
Examples are community maintained, if you think adding an example for react-flow is useful then open a PR and we'll review it when we have the time 👍 .
